frontend/src/components/elements/service/index.jsx

```diff
@@ -1,57 +1,61 @@
 import React from "react";
 import "font-awesome/css/font-awesome.min.css";
-import { ServiceIcon } from "./styles";
+import {ServiceIcon} from "./styles";
```

Comment:
Looks like you have not run prettier

Reply:
But I did run prettier and that's why there's change here 🤔

Reply:
Have you used the prettier config from cthit/style-guides?

Reply:
Yeah. Fixed it now. I formatted with ctrl+shift+alt+L but I should have used ctrl+shift+alt+P heheh